templates/default/rt/rtnoteadd.html

```diff
@@ -67,15 +63,15 @@ <H1>{$layout.pagetitle}</H1>
     <TBODY>
 	<TR>
 		<TD class="bold nobr">
-			<IMG src="img/mail.gif" alt=""> <label for="notify">{trans("Notify users:")}</label>
+			<i class="lms-ui-icon-mail"></i><label for="notify"> {trans("Notify users:")}</label>
 		</TD>
```

Comment:
Jak powyżej. Wszędzie to do poprawy.

Reply:
A przepraszam: tu trzeba pełną nazwę klasy użyć - bo to element HTML `i` bezpośrednio użyty. Jest więc dobrze!